### DISCUSSION TITLE
The place on a wall example model disappears or refuses to be placed on a white wall.

### DISCUSSION THREAD

**Discussion by dbbrooks-lgtm** [google/model-viewer#2240] [Q&A]
### Description
Devices with repro: Samsung S8+,  Pixel 4 XL

Go to https://modelviewer.dev/examples/augmentedreality/#wall and click the enter AR button.

The model does not appear and cannot be placed if most of the camera view is a blank white wall. After pointing the camera at an object on the wall like a painting, the model can be placed. Place the model and then try to move it onto the white wall, where the white wall will take up most of the camera view. The model ends up disappearing when moved into the white space, and it's resistant to being dragged into the white space. 

Screen recording:

https://user-images.githubusercontent.com/77503048/104785598-bbc8fe80-573f-11eb-9d33-acbf40bbeb7e.mp4

### Version
- model-viewer: vX.X.X

### Browser Affected

- [x] Chrome, version 87.0.4280.141
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari
### OS

- [x] Android P (9), Android S (12)
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
This is just a limitation of ARCore, and therefore WebXR. If the surface is too blank, it can't figure out where it is.

**Comment by canberka** [comment]
This is common issue of AR systems. AR is working with camera RGB resolution. So its not only white wall issue. if you try on wall that is red color, same issue will appear .. Except for Iphone 12 Pro Max. It also work with Lidar which has a distance sensor.
